lib/web_ui/lib/src/ui/canvas.dart

```diff
@@ -1068,7 +1068,20 @@ class Picture {
   ///
   /// Although the image is returned synchronously, the picture is actually
   /// rasterized the first time the image is drawn and then cached.
-  Future<Image> toImage(int width, int height) => null;
+  Future<Image> toImage(int width, int height) async {
+    final engine.BitmapCanvas canvas = engine.BitmapCanvas(Rect.fromLTRB(0, 0, width.toDouble(), height.toDouble()));
```

Comment:
Is this supposed to return the full contents of the canvas, resized to width x height? Or is it supposed to clip the canvas contents? It looks like this may clip the canvas contents since the BitmapCanvas you create may not be big enough to hold the contents of the recording canvas.

Reply:
Spoke offline. It seems like width and height are supposed to be the bounds of the canvas that the picture is drawn into.

Reply:
It will clip. I updated the docs.